lib/c-api/Cargo.toml

```diff
@@ -78,14 +78,20 @@ compiler-headless = [
 ]
 singlepass = [
     "wasmer-compiler-singlepass",
+    "wasmer-wasi/emscripten_singlepass", 
+    "wasmer-wasi/wasi_singlepass",
     "compiler",
 ]
 cranelift = [
     "wasmer-compiler-cranelift",
+    "wasmer-wasi/emscripten_cranelift", 
+    "wasmer-wasi/wasi_cranelift",
     "compiler",
 ]
 llvm = [
     "wasmer-compiler-llvm",
+    "wasmer-wasi/emscripten_llvm", 
+    "wasmer-wasi/wasi_llvm",
```

Comment:
This is not correct

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3483/commits/a483d047de82c89a295d3fbf7ce813dfdea1ed32